src/python/pants/core_tasks/clean.py

```diff
@@ -28,7 +28,24 @@ def register_options(cls, register):
                   'for large pants workdirs.')
 
   def execute(self):
+    # If `pants_workdir` is a symlink, use its target. We assume here that we
```

Comment:
`target` may be overloaded in pants codebase to denote things like `jvm_binary` or `python_library`, do you think whether something like `destination` would be appropriate would be?

Reply:
@wisechengyi I meant the target of the symlink, but good point. Per @ity's advice, we're looking to merge a more targeted fix that she's working on. If we decide for some reason to revisit this, I'll be sure to make the change. Thanks!